browser/extensions/manifest_v2/brave_hosted_extensions.h

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_EXTENSIONS_MANIFEST_V2_BRAVE_HOSTED_EXTENSIONS_H_
+#define BRAVE_BROWSER_EXTENSIONS_MANIFEST_V2_BRAVE_HOSTED_EXTENSIONS_H_
+
+#include <array>
+#include <optional>
+
+#include "base/containers/fixed_flat_map.h"
+#include "extensions/common/extension_id.h"
+
+namespace extensions_mv2 {
+
+inline constexpr char kNoScriptId[] = "bgkmgpgeempochogfoddiobpbhdfgkdi";
+inline constexpr char kUBlockId[] = "jcokkipkhhgiakinbnnplhkdbjbgcgpe";
+inline constexpr char kUMatrixId[] = "fplfeajmkijmaeldaknocljmmoebdgmk";
+inline constexpr char kAdGuardId[] = "ejoelgckfgogkoppbgkklbbjdkjdbmen";
+
+inline constexpr char kCwsNoScriptId[] = "doojmbjmlfjjnbmnoijecmcbfeoakpjm";
+inline constexpr char kCwsUBlockId[] = "cjpalhdlnbpafiamejdnhcphjbkeiagm";
+inline constexpr char kCwsUMatrixId[] = "ogfcmafjalglgifnmanfmnieipoejdcf";
+inline constexpr char kCwsAdGuardId[] = "gfggjaccafhcbfogfkogggoepomehbjl";
+
+inline constexpr auto kBraveHosted =
+    base::MakeFixedFlatMap<std::string_view, std::string_view>(
+        {{kNoScriptId, kCwsNoScriptId},
+         {kUBlockId, kCwsUBlockId},
+         {kUMatrixId, kCwsUMatrixId},
+         {kAdGuardId, kCwsAdGuardId}});
+
+inline constexpr auto kCwsHosted =
```

Comment:
Nit: We tend to fail at this a bit at brave but the style guide actually discourage shortening names like that, and I think for this particular case we should avoid it.

> Minimize the use of abbreviations that would likely be unknown to someone outside your project (especially acronyms and initialisms). Do not abbreviate by deleting letters within a word. When an abbreviation is used, prefer to capitalize it as a single "word", e.g., StartRpc() rather than StartRPC(). As a rule of thumb, an abbreviation is probably OK if it's listed in Wikipedia. Note that certain universally-known abbreviations are OK, such as i for a loop index and T for a template parameter.

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules